### PR TITLE
Minor documentation updates for exposing membershipType in decision item

### PR DESCRIPTION
**Pull request by shubhamguptacal**
Schema update PR: 5565546

### CODE REVIEW

File: api-reference/beta/resources/accessreviewinstancedecisionitem.md [hunk 1]
```diff
@@ -48,6 +48,8 @@ Represents an Azure AD [access review](accessreviewsv2-overview.md) decision on
 |resourceLink|String|A link to the resource. For example, `https://graph.microsoft.com/v1.0/servicePrincipals/c86300f3-8695-4320-9f6e-32a2555f5ff8`. Supports `$select`. Read-only.|
 |reviewedBy|[userIdentity](../resources/useridentity.md)| The identifier of the reviewer. Supports `$select`. Read-only.|
 |reviewedDateTime|DateTimeOffset| The timestamp when the review decision occurred. Supports `$select`. Read-only.|
+|principalResourceMembership|[decisionItemPrincipalResourceMembership](../resources/decisionItemPrincipalResourceMembership.md)| Every decision item in an access review represents a principal's membership to a resource. This property represents details of the membership. Examples of this detail might be whether the principal has direct access or indirect access. Supports `$select`. Read-only.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/accessreviewinstancedecisionitem.md:51]:
```suggestion
|principalResourceMembership|[decisionItemPrincipalResourceMembership](../resources/decisionItemPrincipalResourceMembership.md)| Every decision item in an access review represents a principal's membership to a resource. This property provides the details of the membership. For example, whether the principal has direct access or indirect access to the resource. Supports `$select`. Read-only.|
```

File: api-reference/beta/resources/decisionItemPrincipalResourceMembership.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: "decisionItemPrincipalResourceMembership resource type"
+description: "Every decision item in an access review represents a principal's access to a resource. decisionItemPrincipalResourceMembership represents the resource associated with the decision item."
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/decisionItemPrincipalResourceMembership.md:3]:
```suggestion
description: "Each decision in an access review represents a principal's access to a resource. A decisionItemPrincipalResourceMembership object exposes the details of the type of membership that the principal has to the resource that's associated with the accessReviewInstanceDecisionItem object."
```

File: api-reference/beta/resources/decisionItemPrincipalResourceMembership.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+---
+
+# decisionItemPrincipalResourceMembership resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]
+
+Every decision item in an access review represents a principal's membership's to a resource. This property represents details of the membership. Examples of this detail might be whether the principal has direct access or indirect access. To start with, this type has one property : membershipType which indicates the types of membership the principal has to the resource. Types of these memberships includes direct and indirect. In the future, it may include details like when this membership(link) was created, etc.
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/decisionItemPrincipalResourceMembership.md:17]: Let's avoid teasing about what might be included in the future :)
```suggestion
Each decision in an access review represents a principal's access to a resource. A **decisionItemPrincipalResourceMembership** object exposes the details of the type of membership that the principal has to the resource that's associated with the [accessReviewInstanceDecisionItem](accessreviewinstancedecisionitem.md) object. For example, the principal can have direct or indirect access to the resource.
```

File: api-reference/beta/resources/decisionItemPrincipalResourceMembership.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|membershipType|microsoft.graph.decisionItemPrincipalResourceMembershipType|Type of membership. Types include: direct and indirect.|
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/decisionItemPrincipalResourceMembership.md:23]:
```suggestion
|membershipType| decisionItemPrincipalResourceMembershipType | Type of membership that the principal has to the resource. Multi-valued. The possible values are: `direct`, `indirect`, `unknownFutureValue`.|
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 5]
```diff
@@ -663,6 +663,40 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added the **decisionItemPrincipalResourceMembershipType** enumeration type.",
+          "Target": "decisionItemPrincipalResourceMembershipType"
+        },
+        {
+          "Id": "8b32b549-7dd3-46e4-b83e-78f77dd7245e",
+          "ApiChange": "Resource",
+          "ChangedApiName": "decisionItemPrincipalResourceMembership",
+          "ChangeType": "Addition",
+          "Description": "Added the [decisionItemPrincipalResourceMembership](https://docs.microsoft.com/en-us/graph/api/resources/decisionItemPrincipalResourceMembership?view=graph-rest-beta) resource type.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:682]:
```suggestion
          "Description": "Added the [decisionItemPrincipalResourceMembership](https://docs.microsoft.com/en-us/graph/api/resources/decisionItemPrincipalResourceMembership?view=graph-rest-beta) resource type. This object exposes the details of the type of membership that the principal has to the resource that's associated with the [accessReviewInstanceDecisionItem](accessreviewinstancedecisionitem.md) object.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 6]
```diff
@@ -663,6 +663,40 @@
...
       "SubArea": "Governance"
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "ef441c70-5467-4012-ba7b-e40e86069b11",
+          "ApiChange": "EnumType",
+          "ChangedApiName": "decisionItemPrincipalResourceMembershipType",
+          "ChangeType": "Addition",
+          "Description": "Added the **decisionItemPrincipalResourceMembershipType** enumeration type.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:674]:
```suggestion
          "ApiChange": "Enum type",
          "ChangedApiName": "decisionItemPrincipalResourceMembershipType",
          "ChangeType": "Addition",
          "Description": "Added the **decisionItemPrincipalResourceMembershipType** enumeration type.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 7]
```diff
@@ -663,6 +663,40 @@
...
+          "Description": "Added the **principalResourceMembership** property to the [accessReviewInstanceDecisionItem](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItem?view=graph-rest-beta) resource.",
+          "Target": "accessReviewInstanceDecisionItem"
+        }
+      ],
+      "Id": "2c08a551-95b2-43f7-a947-4afe08ea917b",
+      "Cloud": "prd",
+      "Version": "beta",
+      "CreatedDateTime": "2022-01-12T19:40:43.984Z",
+      "WorkloadArea": "Identity and access",
+      "SubArea": "Governance"
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:699]: Please move this changelog record to first in this file: from Line 3.

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @shubhamguptacal for your work on this PR. I've requested for some minor changes to be addressed prior to approval. Also, please move your changelog record to first in the changelog file (from Line 3).

**Comment by FaithOmbongi** [comment]
Light bump @shubhamguptacal, @isabelleatmsft - this PR is getting stale.

**Comment by FaithOmbongi** [comment]
Another bump, @isabelleatmsft, @shubhamguptacal  - the schema related to this docs PR has been live for a long time now.

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json (hunk 6 above)
Comment (zhusijia26) [reply changelog/Microsoft.IdentityGovernance.AccessReviews.json:674]: @shubhamguptacal I am afraid this change does not work. Could you please comment?

**Review by zhusijia26 — APPROVED** [review]

**Review by zhusijia26 — APPROVED** [review]

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**